service/src/test/java/greencity/service/ubs/UBSManagementServiceImplTest.java

```diff
@@ -738,8 +733,6 @@ void testGetOrderDetails() {
         }.getType());
         verify(bagRepository).findBagByOrderId(1L);
         verify(modelMapper).map(TEST_BAG, BagInfoDto.class);
-        verify(bagTranslationRepository).findAllByOrder(1L);
```

Comment:
we should not remove it at all
please use verification for  `bagRepository.findAllByOrder`

Reply:
add this